lib/components/Widgets/Dashboard/index.stories.tsx

```diff
@@ -18,7 +18,8 @@ const renderWidget = (index: number) => {
     () => <AreaChartWidget {...AreaChartWidgetStoriesMeta.args} />,
     () => (
       <WidgetContainer
-        header={{ title: "A form widget", description: "Enter your data" }}
+        header={{ title: "A form widget" }}
+        description="Enter your data"
```

Comment:
There's something about this API that makes me a bit uncomfortable. I understand how having `header` separately groups things logically, but it also somehow bleeds implementation details.

My question is: Do we really have a use case for `description`? Should we just remove it?